lib/converter.rb

```diff
@@ -0,0 +1,51 @@
+class Converter
+
+  def number_to_letters(number)
+    letter_lookup = {
+      1 => "  A ",
+      2 => "  A B ",
+      3 => "  A B C ",
+      4 => "  A B C D ",
+      5 => "  A B C D E ",
+      6 => "  A B C D E F ",
+      7 => "  A B C D E F G ",
+      8 => "  A B C D E F G H ",
+      9 => "  A B C D E F G H I ",
+      10 => "  A B C D E F G H I J "
+    }
+    letter_lookup[number]
+  end
+
+  def number_to_array_position(number)
```

Comment:
Perhaps you could have a method that converts the number by converting to a string and subtracting 1, instead of using a hash. This way no matter how many numbers might exist in your game in the future this method would not have to change.